securitas/tests/unit/conftest.py

```diff
@@ -52,18 +52,49 @@ def vcr_cassette_dir(request):
 
 
 @pytest.fixture
-def dummy_user():
-    try:
+def make_user():
+    created_users = []
+
+    def _make_user(name):
+        now = datetime.datetime.utcnow().replace(microsecond=0)
+        password = f'{name}_password'
         ipa_admin.user_add(
-            'dummy',
-            'Dummy',
+            name,
+            name.title(),
             'User',
-            'Dummy User',
-            user_password='dummy_password',
+            f'{name.title()} User',
+            user_password=password,
             login_shell='/bin/bash',
+            fascreationtime=f"{now.isoformat()}Z",
         )
         ipa = untouched_ipa_client(app)
-        ipa.change_password('dummy', 'dummy_password', 'dummy_password')
-        yield
-    finally:
-        ipa_admin.user_del('dummy')
+        ipa.change_password(name, password, password)
```

Comment:
Question - why is this line necessary? Is it a temporary password if created using the API?

Reply:
When a user is create with the API, it's created with an expired password, so they are forced to change it before they can login. We don't want that, so we're changing the password ourselves. It's also what the registration controller does.